app/components/views/Offset/index.tsx

```diff
@@ -464,54 +464,50 @@ export const Offset = (props: Props) => {
             labelAlignment="start"
           />
 
-          <div className={styles.input}>
-            <label>
-              <Text t="caption" color="lightest">
-                <Trans id="offset.beneficiary">BENEFICIARY</Trans>
-              </Text>
-            </label>
-            <input
-              value={beneficiary}
-              onChange={(e) => {
-                setBeneficiary(e.target.value);
-              }}
-              placeholder={t({
-                id: "offset.who_beneficiary",
-                message: "To whom will this retirement be credited?",
-              })}
-            />
-          </div>
-          <div className={styles.input}>
-            <label>
-              <Text t="caption" color="lightest">
-                <Trans id="offset.beneficiary_address">
-                  BENEFICIARY ADDRESS (optional: defaults to connected address)
+          <div className={styles.beneficiary}>
+            <Text t="caption" color="lighter">
+              <Trans id="offset.retirement_message">
+                Who will this retirement be credited to?
+              </Trans>
+            </Text>
+            <div className={styles.input}>
+              <input
+                value={beneficiary}
+                onChange={(e) => setBeneficiary(e.target.value)}
+                placeholder={t({
+                  id: "offset.beneficiary",
+                  message: "Name or organisation",
+                })}
+              />
+            </div>
+
+            <div className={styles.input}>
+              <input
+                value={beneficiaryAddress}
+                onChange={(e) => setBeneficiaryAddress(e.target.value)}
+                placeholder={t({
+                  id: "offset.ethereum_address",
+                  message: "Ethereum address",
+                })}
+              />
+              <Text t="caption" color="lightest" className="defaultAddress">
+                <Trans id="offset.default_retirement_address_message">
+                  Defaults to the connected wallet address
                 </Trans>
               </Text>
-            </label>
-            <input
-              value={beneficiaryAddress}
-              onChange={(e) => {
-                setBeneficiaryAddress(e.target.value);
-              }}
-              placeholder={t({
-                id: "offset.which_address_retiring",
-                message: "Which address are you retiring on behalf of?",
-              })}
-            />
+            </div>
           </div>
+
           <div className={styles.input}>
             <label>
```

Comment:
I know this isn't your doing but since we in here:
Do you think screenreaders or other a11y tools benefit from adding an `id` here along with a `for` on the TextArea? I'm getting rusty on a11y already

Reply:
Absolutely, we have placeholder text that will help with a11y in lieu of having explicit labels.
Thinking of addressing this in the next piece of pledge dashboard work, where I'll create some reusable form input components. We can retrospectively use those form components in that this form.

What do you think? :)

https://www.a11ywithlindsey.com/blog/introduction-accessible-labeling